import-scripts/subset-impact-data.sh

```diff
@@ -55,6 +55,26 @@ if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
 fi
 echo -e "\tPORTAL_SCRIPTS_DIRECTORY="$PORTAL_SCRIPTS_DIRECTORY
 
+# check for presence of new filename pattens (not yet supported) and fail if present
+
+PYTHON_FILENAME_CHECK_CALL="import sys
```

Comment:
switch to our usual convention of calling python scripts (see fetch-dmp-data)

Reply:
done